provider/provider.go

```diff
@@ -94,3 +94,17 @@ func getDefaultPath() string {
 		return "/usr/local/lib/summon"
 	}
 }
+
+// GetAllProviders creates slice of all file names in the default path
+func GetAllProviders(providerDir string) ([]string, error){
+	files, err := ioutil.ReadDir(providerDir)
+	if err != nil{
```

Comment:
You might want to run your code through `go lint` - I think there are some failures that might not be apparent.

Reply:
I have, but it's not failing on my changes

Reply:
hmm.. the spacing in this line is not considered an issue? I'll run a checker locally - I would imagine that the `nil{` should be `nil {`